src/test/ui/imports/glob-resolve1.stderr

```diff
@@ -4,21 +4,17 @@ error[E0425]: cannot find function `fpriv` in this scope
 LL |     fpriv();
    |     ^^^^^ not found in this scope
    |
-help: consider importing this function
-   |
-LL | use bar::fpriv;
-   |
+   = note: this function exists but is inaccessible:
+           bar::fpriv
```

Comment:
For the case where there's a single function being mentioned, we should not present the "list" output, instead show "function `bar::fpriv` exists but is inaccessible".